backend/app/processing/free_llm_formatter.py

```
@@ -0,0 +1,80 @@
+import requests
+from typing import Dict, Any, Optional
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)